app/src/main/java/org/wikipedia/readinglist/ReadingListBehaviorsUtil.kt

```diff
@@ -146,14 +148,17 @@ object ReadingListBehaviorsUtil {
         if (lists == null) {
             return
         }
-        FeedbackUtil
-                .makeSnackbar(activity,
-                        activity.getString(
-                                if (lists.size == 1) R.string.reading_list_item_deleted else R.string.reading_lists_item_deleted, page.title()),
-                        FeedbackUtil.LENGTH_DEFAULT)
+        val readingListNames = mutableListOf<String>()
+        lists.forEach { readingList ->
+            readingListNames.add(readingList.title())
+        }
```

Comment:
Would it be possible to simplify as below and not running the logic in the snackbar?
```
        val readingListNames = lists.map { it.title() }.run {
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
                ListFormatter.getInstance().format(this)
            } else {
                joinToString(separator = ", ")
            }
        }
```